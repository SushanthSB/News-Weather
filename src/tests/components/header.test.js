import React from 'react';
import Header from '../../components/header'
import renderer from 'react-test-renderer'

it('Heder should render without breaking', () => {
	const tree = renderer.create(
			<nav className='navbar navbar-default navbar-fixed-top'>
						<h1 className='brand'>Daily News</h1>
			</nav>
		).toJSON();
	expect(tree).toMatchSnapshot();
})