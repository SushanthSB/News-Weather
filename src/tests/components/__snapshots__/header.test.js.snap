// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heder should render without breaking 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-top"
>
  <h1
    className="brand"
  >
    Daily News
  </h1>
</nav>
`;
